targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/TARGET_MTB_LAIRD_BL654/PinNames.h

```diff
@@ -0,0 +1,315 @@
+	PinDef(1 , 13),
+	PinDef(1 , 14),
+	PinDef(1 , 15),
+    
+    // Port0
+    p0  = P0_0,
+    p1  = P0_1,
+    p2  = P0_2,
+    p3  = P0_3,
+    p4  = P0_4,
+    p5  = P0_5,
+    p6  = P0_6,
+    p7  = P0_7,
+
+    p8  = P0_8,
+    p9  = P0_9,
+    p10 = P0_10,
+    p11 = P0_11,
+    p12 = P0_12,
+    p13 = P0_13,
+    p14 = P0_14,
+    p15 = P0_15,
+
+    p16 = P0_16,
+    p17 = P0_17,
+    p18 = P0_18,
+    p19 = P0_19,
+    p20 = P0_20,
+    p21 = P0_21,
+    p22 = P0_22,
+    p23 = P0_23,
+
+    p24 = P0_24,
+    p25 = P0_25,
+    p26 = P0_26,
+    p27 = P0_27,
+    p28 = P0_28,
+    p29 = P0_29,
+    p30 = P0_30,
+    p31 = P0_31,
+    
+    // Port1
+    p32 = P1_0,
+    p33 = P1_1,
+    p34 = P1_2,
+    p35 = P1_3,
+    p36 = P1_4,
+    p37 = P1_5,
+    p38 = P1_6,
+    p39 = P1_7,
+
+    p40 = P1_8,
+    p41 = P1_9,
+    p42 = P1_10,
+    p43 = P1_11,
+    p44 = P1_12,
+    p45 = P1_13,
+    p46 = P1_14,
+    p47 = P1_15,
+
+    // Not connected
+    NC = (int)0xFFFFFFFF,
+
+    SIO_0 = P0_0,
+    SIO_1 = P0_1,
+    SIO_2 = P0_2,
+    SIO_3 = P0_3,
+    SIO_4 = P0_4,
+    SIO_5 = P0_5,
+    SIO_6 = P0_6,
+    SIO_7 = P0_7,
+    SIO_8 = P0_8,
+    SIO_9 = P0_9,   //NFC1
+    SIO_10 = P0_10,  //NFC2
+    SIO_11 = P0_11,
+    SIO_12 = P0_12,
+    SIO_13 = P0_13,
+    SIO_14 = P0_14,
+    SIO_15 = P0_15,
+
+    SIO_16 = P0_16,
+    SIO_17 = P0_17,
+    SIO_18 = NC,
+    SIO_19 = P0_19,
+    SIO_20 = P0_20,
+    SIO_21 = P0_21,
+    SIO_22 = P0_22,
+    SIO_23 = P0_23,
+    SIO_24 = P0_24,
+    SIO_25 = P0_25,
+    SIO_26 = P0_26,
+    SIO_27 = P0_27,
+    SIO_28 = P0_28,
+    SIO_29 = P0_29,
+    SIO_30 = P0_30,
+    SIO_31 = P0_31,
+ 
+    SIO_32 = P1_0,
+    SIO_33 = P1_1,
+    SIO_34 = P1_2,
+    SIO_35 = P1_3,
+    SIO_36 = P1_4,
+    SIO_37 = P1_5,
+    SIO_38 = P1_6,
+    SIO_39 = P1_7,
+    SIO_40 = P1_8,
+    SIO_41 = P1_9,
+    SIO_42 = P1_10,
+    SIO_43 = P1_11,
+    SIO_44 = P1_12,
+    SIO_45 = P1_13,
+    SIO_46 = P1_14,
+    SIO_47 = P1_15,
+
+    LED1    = SIO_38,
+    LED2    = SIO_39,
+    LED3    = SIO_37,
+    LED_RED = LED1,
+    LED_GREEN = LED3,
+    LED_BLUE = LED2,
+
+    BUTTON1 = SIO_33,
+    USER_BUTTON = BUTTON1,
+
+    //Nordic SDK pin names 
+    RX_PIN_NUMBER  = SIO_8,
+    TX_PIN_NUMBER  = SIO_6,
+    CTS_PIN_NUMBER = SIO_7,
+    RTS_PIN_NUMBER = SIO_5,
+
+    // mBed interface Pins
+    USBTX = TX_PIN_NUMBER,
+    USBRX = RX_PIN_NUMBER,
+    STDIO_UART_TX = TX_PIN_NUMBER,
+    STDIO_UART_RX = RX_PIN_NUMBER,
+    STDIO_UART_CTS = CTS_PIN_NUMBER,
+    STDIO_UART_RTS = RTS_PIN_NUMBER,
+
+    SPI_PSELMOSI0 = SIO_45,
+    SPI_PSELMISO0 = SIO_46,
+    SPI_PSELSS0   = SIO_42, //CS for SD card on MTB
+    SPI_PSELSCK0  = SIO_47,
+
+    SPI_PSELMOSI1 = SIO_30,
+    SPI_PSELMISO1 = SIO_29,
+    SPI_PSELSS1   = SIO_41, //CS for LCD on MTB
+    SPI_PSELSCK1  = SIO_44,
+
+    //Default SPI
+    SPI_MOSI = SPI_PSELMOSI0,
+    SPI_MISO = SPI_PSELMISO0,
+    SPI_SCK = SPI_PSELSCK0,
+    SPI_CS = SPI_PSELSS0,
+
+/*
```

Comment:
why are these commented out?

Reply:
@0xc0170 : Thanks. Because the SPIS pins are not brought out on the MCB/MTB .. I kept it there (in comments) so that it may help end users to add these if they decide to bring these pins out from the module on their respective boards..